frontend/src/context/stepper/StepperSlice.tsx

```diff
@@ -2,18 +2,29 @@ import { createSlice } from '@reduxjs/toolkit';
 import { ZERO } from '@src/constants/commons';
 import type { RootState } from '@src/store';
 
+export interface IMetricsPageFailedDateRange {
+  boardInfoError?: boolean;
+  pipelineInfoError?: boolean;
+  pipelineStepError?: boolean;
```

Comment:
is/hasBoardInfoError......
why it is `?`? the default value is true or false when not give the specific value when implamentation

Reply:
There is something really wrong with the name. But the value needs to be '?', so that the subsequent assignment will be more convenient